pkg/controller/user-cluster-controller-manager/resources/controller.go

```diff
@@ -60,9 +60,10 @@ const (
 )
 
 type UserClusterMLA struct {
-	Logging       bool
-	Monitoring    bool
-	MLAGatewayURL string
+	Logging                               bool
+	Monitoring                            bool
+	MLAGatewayURL                         string
+	PrometheusCustomScrapeConfigMapPrefix string
```

Comment:
This name is also quite long. Maybe just `CustomScrapeConfigPrefix` or something like that would be better.

Reply:
How about we drop `Custom` and `Map`, so it will be `PrometheusScrapeConfigPrefix`? I would like to have `Prometheus` in the name, because we may have the same thing for `Loki` (for logs) in the future, for example, `LokiScrapeConfigPrefix`

Reply:
Sounds good. :+1:

Reply:
Fixed :)